### ISSUE TITLE
NTR: Glutarylglycine high in urine (Lab: urine tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8803]
New term request Glutarylglycine high in urine (segal_030514192718):

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000129
- label: Elevated urinary glutarylglycine level
- parents: [Elevated urinary carboxylic acid (HP:0040156)](https://hpo.jax.org/app/browse/term/HP:0040156)
- pmid: 
- def: The amount of glutarylglycine in the urine, normalized for urine concentration, is below the lower limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8803
- synonyms:
